### DISCUSSION TITLE
Typo3 12.4.8 and Extension Builder 11.0.9

### DISCUSSION THREAD

**Discussion by bogdan-ciocoiu** [FriendsOfTYPO3/extension_builder#665] [Q&A] [answered]
Has anyone else tried to install **Extension Builder 11.0.9** on **Typo3 12.4.8**? When I try to activate the former, I get:

```
The following errors were found while trying to install "extension_builder":
Your TYPO3 version is higher than this extension requires. It requires TYPO3 versions 11.4.0 - 11.9.99
```

**Comment by DavidBruchmann** [comment, marked as answer]
It won't work as it's intended for TYPO3 v11.
I suppose this is the right version for v12: https://github.com/FriendsOfTYPO3/extension_builder/tree/v12.0.0-alpha.1

**Comment by PKuhlmay** [comment]
As David wrote, version 11 of the extension_builder is only for TYPO3 11. v12.0.0-alpha.1 is for TYPO3 12 but it is not finished yet. It may contain bugs. It is only available via composer at the moment. I cannot upload new version to the TER at the moment and alpha-versions are not allowed there, too.
